Microsoft.Toolkit.Uwp.UI.Controls/ImageCropper/ImageCropper.Properties.cs

```diff
@@ -0,0 +1,171 @@
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Media.Imaging;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// The <see cref="ImageCropper"/> control allows user to crop image freely.
+    /// </summary>
+    public partial class ImageCropper
+    {
+        /// <summary>
+        /// Gets or sets the minimum cropped length(in pixel).
+        /// </summary>
+        public double MinCroppedPixelLength { get; set; } = 40;
+
+        /// <summary>
+        /// Gets or sets the minimum selectable length.
+        /// </summary>
+        public double MinSelectedLength { get; set; } = 40;
+
+        private static void OnSourceChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            var target = (ImageCropper)d;
+            if (e.NewValue is WriteableBitmap bitmap)
+            {
+                if (bitmap.PixelWidth > target.MinCropSize.Width && bitmap.PixelHeight > target.MinCropSize.Height)
+                {
+                    target.InitImageLayout();
+                }
+                else
+                {
+                    throw new ArgumentException("The resolution of the image is too small!");
+                }
+            }
+        }
+
+        private static void OnAspectRatioChanged(
+            DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            var target = (ImageCropper)d;
+            target.UpdateAspectRatio();
+        }
+
+        private static void OnCircularCropChanged(
+            DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            var target = (ImageCropper)d;
+            target.UpdateAspectRatio();
+            target.UpdateThumbsVisibility();
+            target.UpdateMaskArea();
+        }
+
+        private static void OnIsSecondaryControlButtonVisibleChanged(
+            DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            var target = (ImageCropper)d;
+            target.UpdateThumbsVisibility();
+        }
+
+        /// <summary>
+        ///  Gets or sets the source of the cropped image.
+        /// </summary>
+        public WriteableBitmap Source
+        {
+            get { return (WriteableBitmap)GetValue(SourceProperty); }
+            set { SetValue(SourceProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the aspect ratio of the cropped image，the default value is -1.
+        /// </summary>
+        public double AspectRatio
+        {
+            get { return (double)GetValue(AspectRatioProperty); }
+            set { SetValue(AspectRatioProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether gets or sets whether to use a circular ImageCropper.
+        /// </summary>
+        public bool CircularCrop
```

Comment:
Probably would be better to make this an enum
`public enum CropShape { Rectangular, Circular }`.
This allows for other options in the future

Reply:
Yes, although I think most of the time there may not be other options, it's better.